### DISCUSSION TITLE
Wildcard custom domains (not subdomain)

### DISCUSSION THREAD

**Discussion by ralphilius** [vercel/vercel#4721] [Help] [answered]
Hello,

I want to dynamically add multiple domains to one of my project. Is there any way to achieve this?

Thank you 🙏

**Comment by paulogdm** [comment, marked as answer]
Hello,

You can do that by using the [Vercel api](api.vercel.com). Do notice you do have [limitations](https://vercel.com/docs/v2/platform/limits) on the number of domains per-project.
